sky/clouds/service_catalog/data_fetchers/fetch_gcp.py

```diff
@@ -138,7 +140,7 @@ def _get_unit_price(sku: Dict[str, Any]) -> float:
     return units + nanos
 
 
-@functools.lru_cache(maxsize=None)
+@functools.lru_cache(maxsize=1)
```

Comment:
Just curious: what's the reason for this change?

Reply:
Ahh, it is a remnant. Changed it back. Thanks!